rs/nervous_system/common/src/lib.rs

```diff
@@ -83,6 +83,15 @@ const WASM_PAGE_SIZE_BYTES: usize = 65536;
 // 1 Mi. Approximately 10^6, 1 million (slightly more).
 const MAX_LOGS_RESPONSE_SIZE: usize = 1 << 20;
 
+/// Generates as many user test ids as requested, starting from `starting_index`.
+/// ```rust
+/// // generate 3 test IDs, with indices starting at 55:
+/// let [id1, id2, id3] = new_user_test_ids(55);
+/// ```
+pub fn new_user_test_ids<const N: usize>(starting_index: u64) -> [PrincipalId; N] {
```

Comment:
Is this supposed to be a test-only function? If so, let's not declare it in this crate. At the very least, test functions should be compiled only for tests.

Reply:
I removed the function